tests/cts_tests/interpreter/add_f.shr

```diff
@@ -0,0 +1,12 @@
+lda.imm.f 15
+mov.imm.f r0, -15
+mov.imm.f r1, 0
+add.f r0
+ftoi32
+jump.eq r1, label_2
+label_1:
+lda.imm.i32 1
+ret
+label_2:
```

Comment:
Maybe add tabulation for code after labels

Reply:
I think about code-style like this:
`function main (i32 a0) {`
`  ret`
`label:`
`  ret`
`}`

But we will be able to implement this code-style after function implementation